kafka-ui-react-app/src/components/common/Input/Input.tsx

```diff
@@ -181,8 +184,11 @@ const Input: React.FC<InputProps> = ({
           type={type}
           onKeyPress={keyPressEventHandler}
           onPaste={pasteEventHandler}
+          ref={ref}
           {...inputOptions}
         />
+        {search && children}
```

Comment:
i don't think passing it as `children` here is very intuitive , we can pass it as a prop , because i think it will be hard to understand what is a `children`  of a `Input` component.

Reply:
ok i will try to change